Specify directories for test files in testing.md (#262)
rust-cli/book: src/tutorial/testing.md

}
```

> You can put this snippet of code in pretty much any file in `src/` or `tests/`

Reviewer: > The current wording might suggest, any arbitrary location for test files.
But, cargo limits it's search to src/ and test/

I feel like this is overly prescriptive and trying to decide how to balance this.  One idea is

```suggestion
You can put this snippet of code in pretty much any source file in your package
```
